clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/config/DockerRegistryConfigurationProperties.groovy

```diff
@@ -33,6 +33,8 @@ class DockerRegistryConfigurationProperties {
     String email
     // Address of the registry.
     String address
+    // How many threads to cache all provided repos on. Really only useful if you have a ton of repos.
+    int cacheThreads
```

Comment:
You'll need to add this field to your clouddriver.yml to get this to work. Try playing with the number to see what happens. (Maybe 100 for starters? Don't know.)